src/examples/login/Login.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import { action } from "@storybook/addon-actions";
+import type { Meta, StoryObj } from "@storybook/react";
+import { Button } from "../../components/Button/Button";
+import { Checkbox } from "../../components/Checkbox/Checkbox";
+import { Heading } from "../../components/Heading/Heading";
+import { Link } from "../../components/Link/Link";
+
+export default {
+  parameters: {
+    layout: "centered",
+  },
+} satisfies Meta;
+
+export const Default: StoryObj = {
+  render: () => {
+    return (
+      <form
+        onSubmit={(event) => {
+          event.preventDefault();
+          action("on-submit")(event);
+        }}
+        style={{ width: 500 }}
+      >
+        <div style={{ marginBottom: "2rem" }}>
+          <Heading level={1}>안녕하세요, 회원이신가요?</Heading>
+        </div>
+
+        <div
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            gap: "0.5rem",
+            marginBottom: "1rem",
+          }}
+        >
+          <input
+            style={{
+              border: "1px solid lightgray",
+              borderRadius: "0.5rem",
+              paddingLeft: "0.5rem",
+              height: 50,
+            }}
+            required
+            type='email'
+            placeholder='이메일 주소'
+          />
+          <input
+            style={{
+              border: "1px solid lightgray",
+              borderRadius: "0.5rem",
+              paddingLeft: "0.5rem",
+              height: 50,
+            }}
+            required
+            type='password'
+            placeholder='비밀번호'
+          />
+        </div>
+
+        <div
+          style={{
+            marginBottom: "1rem",
+            display: "flex",
+            justifyContent: "space-between",
+          }}
+        >
+          <Link>비밀번호 찾기</Link>
```

Comment:
href 속성이 없는 a 태그는 링크가 아닙니다.

```suggestion
          <Link href="#">비밀번호 찾기</Link>
```

@hyoseong1994 님, `<Link/>` 컴포넌트 설계하실 때 `href` 속성을 필수 입력으로 하지 않았던 이유가 있으션던가요?

Reply:
@hyoseong1994 효성님! 답변 확인하였습니다. 감사합니다 :)

궁금한 사항이 있습니다!
a tag에 href속성없이 onClick을 사용하는 경우가 어떤게 있을까용?
그리고, 만약 href없이 onClick을 사용한다면 굳이 a tag를 사용하는 이유가 있는지 궁금합니다 :)